gradle/libs.versions.toml

```diff
@@ -22,6 +23,7 @@ androidx-constraintlayout = { module = "androidx.constraintlayout:constraintlayo
 androidx-recyclerview = { module = "androidx.recyclerview:recyclerview", version = "1.4.0" }
 androidx-cardview = { module = "androidx.cardview:cardview", version = "1.0.0" }
 material-view = { module = "com.google.android.material:material", version = "1.12.0" }
+material-icons = { module = "org.jetbrains.compose.material:material-icons-core", version.ref = "materialIconsVersion" }
 
```

Comment:
Why do we need this dependency?

Reply:
I think we can remove the dependency as it is not used anywhere in the project itself.